server.js

```diff
@@ -27,6 +27,11 @@ const { logger }      = Server;
 // Need these here so we can run hooks
 logger.add_log_methods(Server, 'server');
 
+// TODO: There was an expressed desire to make these values configurable.
+//       Also, SonarLint said these could be top-level; maybe for re-use?
+const disconnect_timeout = 30;
+const exit_timeout = 30;
```

Comment:
Just because a variable _can_ be top level doesn't mean it _should_. One off variables tend to be best located near the code using them. Maybe you like scrolling up and down pages of code to read a function, but most of us don't. If we want to make them configurable, we'll put them into an .ini file and access them via a cfg object, not global variables. Please revert.

Reply:
I fixed it; due apologies. Clarified that it shouldn't be moved if someone else finds it.